sphinx/writers/latex.py

```diff
@@ -1680,20 +1683,24 @@ def depart_legend(self, node):
     def visit_admonition(self, node):
         # type: (nodes.Element) -> None
         self.body.append('\n\\begin{sphinxadmonition}{note}')
+        self.in_box += 1
```

Comment:
as explained above, it does make sense to prohibit floating figures even if user has special custom LaTeX macros fro admonitions. But ``in_box`` is a bit confusing because it is not true that default admonition types use TeX boxes : only `warning`, `caution`, ... do.